Jenkinsfile

```diff
@@ -197,27 +205,19 @@ pipeline {
         }
       }
       steps {
-        deleteDir()
-        unstash 'source'
-        dir("${BASE_DIR}"){
-          buildDocs(docsDir: "docs", archive: true)
+        withGithubNotify(context: 'Documentation', tab: 'artifacts') {
```

Comment:
Is this step still needed, seeing as there's the separate elasticsearch-ci docs-build check?

Reply:
That's actually a good point, it's completely duplicated and might mislead. Even though I initially thought it might be interesting to keep in case the notification from elasticsearch-ci docs-build is disabled for whatever reason. So I'll remove that check from now on